### ISSUE TITLE
problems on r-universe

### ISSUE THREAD

**Issue by maelle** [richfitz/rrlite#8]
see https://ropensci.r-universe.dev/builds

apparently an installation problem?

**Comment by maelle** [comment]
@richfitz what's the maintenance status of rrlite? :smile_cat:

**Comment by richfitz** [comment]
I have/had a bit of a pr that fixed some of this a while ago, but the upstream project has stalled and it never worked well on windows I believe...

**Comment by maelle** [comment]
Is this package still used? Do you think we should try to get more contributors to it, or should we archive it? Thank you!

**Comment by richfitz** [comment]
I don't think anyone has used it in a long time, and I don't think it ever worked on Windows, but I might be able to get it working again at some point. It would be december at the earliest though!

**Comment by maelle** [comment]
@richfitz do _you_ use it? Thank you in any case!

**Comment by richfitz** [comment]
No, I don't! The lack of windows support and meant I never really found a good use for this, and ended up developing thor instead

**Comment by maelle** [comment]
So why not archive it? Do you expect there are still users of this package?

**Comment by maelle** [comment]
@richfitz should we archive this package?

**Comment by jeroen** [comment]
It no longer installs, I doubt anyone uses this if Rich himself doesn't. Let's archive it.
